src/main/java/messaging/MessageDBGateway.java

```diff
@@ -1,8 +1,22 @@
 package messaging;
 
 import database.RoomDBModel;
-
+/**
+ * The MessageDBGateway interface defines methods to interact with the database related to rooms and messages.
+ */
 public interface MessageDBGateway {
+    /**
+     * Retrieves a RoomDBModel object for the given room ID from the database.
+     *
+     * @param roomID The ID of the room to retrieve from the database.
+     * @return The RoomDBModel object representing the room retrieved from the database.
+     */
     RoomDBModel getARoom(Integer roomID);
+
+    /**
+     * Sends a RoomDBModel object representing a room to the database.
```

Comment:
we no longer have `sendAMessage` method in `MessageDbGateway`. Please check out main branch and edit this document after you resolve merge conflict.